install_scripts/templates/kubernetes/migrate.sh

```diff
@@ -312,6 +302,8 @@ startAppOnK8s() {
 
     logSubstep "restore app config"
     /usr/local/bin/replicatedctl app-config import --skip-validation < "${TMP_DIR}/app-config.json"
+
+    purgeNativeScheduler
```

Comment:
The script exits at the end of this `startAppOnK8s` function if activation is required. The only step that was left was purge native, so I moved it into here so it runs before the script exits.